src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -249,6 +249,20 @@ public static bool IsTrivia(SyntaxKind kind)
             }
         }
 
+        internal static bool IsPreprocessorDirectiveAcceptingPreprocessingSymbols(SyntaxKind kind)
```

Comment:
is this helper used from multiple places, or just a single place?  if a single place, inline to that location.

Reply:
Just this one place, will inline in the same file.